helm/inteli-api/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: inteli-api
-appVersion: '1.7.2'
+appVersion: '1.7.3'
```

Comment:
Please bump to `1.8.0` as this adds new features, rather than being a bugfix

Reply:
Will do, I think that's probably what the conflicts are

Reply:
Sorry I made a mistake, please bump to `1.9.0`